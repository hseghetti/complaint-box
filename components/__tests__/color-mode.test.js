import React from 'react';
import renderer from 'react-test-renderer';
import ColorMode from '../color-mode';

describe('Color Mode Component', () => {
  it('Should render Color Mode', () => {
    const component = renderer.create(
      <ColorMode />
    );
    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});