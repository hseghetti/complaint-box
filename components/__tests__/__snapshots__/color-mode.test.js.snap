// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color Mode Component Should render Color Mode 1`] = `
<button
  className="chakra-button css-oqfhsh"
  type="button"
>
  <svg
    className="chakra-icon css-onkibi"
    focusable={false}
    viewBox="0 0 24 24"
  >
    <g
      fill="none"
      stroke="currentColor"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeWidth="2"
    >
      <circle
        cx="12"
        cy="12"
        r="5"
      />
      <path
        d="M12 1v2"
      />
      <path
        d="M12 21v2"
      />
      <path
        d="M4.22 4.22l1.42 1.42"
      />
      <path
        d="M18.36 18.36l1.42 1.42"
      />
      <path
        d="M1 12h2"
      />
      <path
        d="M21 12h2"
      />
      <path
        d="M4.22 19.78l1.42-1.42"
      />
      <path
        d="M18.36 5.64l1.42-1.42"
      />
    </g>
  </svg>
</button>
`;
